### PR TITLE
Set cache-only next fetch policy for mixed Type feeds

### PR DESCRIPTION
**Pull request by Discordius**

### CODE REVIEW

File: packages/lesswrong/components/common/MixedTypeFeed.tsx
Comment (jimrandomh) [inline packages/lesswrong/components/common/MixedTypeFeed.tsx:180]: Stray `console.log`

**Comment by jimrandomh** [comment]
Note that the bug that this fixes didn't replicate for me when I tried it on prod today, so some aspect of this is still poorly understood. I think this is fine as long as it's merging with the other QA-needed stuff and the QA pass happens.

**Review by jimrandomh — APPROVED** [review]

File: packages/lesswrong/components/common/MixedTypeFeed.tsx
Comment (Discordius) [reply packages/lesswrong/components/common/MixedTypeFeed.tsx:180]:
```suggestion
```

**Comment by Discordius** [comment]
(Waiting until we merged devel into master, but will merge after we did that)

**Pull request merged**
